meson.build

```diff
@@ -46,6 +50,7 @@ features = {
     'avif-muxer': libavformat.version().version_compare('>= 59.24.100'),
     'libass': true,
     'threads': true,
+    'libplacebo': true,
```

Comment:
Hmm, maybe not related, because other are also like that, but why is it useful to set feature always to true?

Reply:
So it's listed in `Message: List of enabled features: [...]` I suppose

Reply:
But they are always enabled, but whatever.